engine/lib/output.php

```diff
@@ -142,6 +142,11 @@ function elgg_format_attributes(array $attrs = array()) {
 	$attributes = [];
 
 	foreach ($attrs as $attr => $val) {
+		if (!is_string($attr) || $attr[0] == '#') {
```

Comment:
> You want the input views to know what the parent field variables are. That's handy for view var filtering.

really? what is the use case? only parent field variables are #type, #help, #label... why would you need those?